models/channel.coffee

```diff
@@ -23,6 +23,9 @@ module.exports = class Channel extends Block
   href: ->
     "/#{@get('user')?.slug}/#{@get('slug')}"
 
+  nMoreBlocks: (n) ->
+    @get('length') - n
```

Comment:
Can you make this method do the correct thing in the case of n being greater than the length?